rocketpy/motors/tank_geometry.py

```diff
@@ -373,6 +400,7 @@ def __init__(self, radius, height, spherical_caps=False, geometry_dict=None):
         """
         geometry_dict = geometry_dict or {}
         super().__init__(geometry_dict)
+        self.__input_radius = radius
```

Comment:
Tough question, I revisited the naming and the issue is that `radius` is a `property` of the parent class. If it were a common method I would think it is a poor name, but it is reasonable for a `property`, but we can discuss changing it. The bigger issue to me is the parameter name being `radius`, which causes the conflict.

For now, I don't think there is another way than creating the `self.__input_radius` attribute. Even if we decide to change the name, it would be necessary a deprecation notice period, which still necessitates the `self.__input_radius`.

Reply:
@phmbressan would you mind opening an issue (or adding a TODO) so we keep following this `radius` naming problem?

Reply:
I have just created #759.